bittensor/utils/__init__.py

```diff
@@ -226,3 +227,12 @@ def create_pow( subtensor, wallet ):
         'block_hash': block_hash, 
         'work': binascii.hexlify(seal)
     }
+
+def version_checking():
+    response = requests.get(f'https://pypi.org/pypi/bittensor/json')
```

Comment:
Why is this an f-string? Also this is probably a good candidate for something that should live in `constants.py` or something similar.

Reply:
Oops, nice catch! Let me add that to init file